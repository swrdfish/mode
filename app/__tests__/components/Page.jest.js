import React from 'react'
import { shallow } from 'enzyme'
import Page from '../../components/Page'
import renderer from 'react-test-renderer';

describe('Component: Page', () => {
    it('should render the page', () => {
        const wrapper = shallow(<Page><span>hello</span></Page>)
        expect(wrapper.find("#main-window")).toHaveLength(1)
        expect(wrapper.find('span')).toHaveLength(1)
        expect(wrapper.find('span').text()).toEqual('hello')
    })

    it('should match previous snapshot', () => {
        const tree = renderer
            .create(<Page><div>hello</div></Page>)
            .toJSON()
        expect(tree).toMatchSnapshot();
    })
})