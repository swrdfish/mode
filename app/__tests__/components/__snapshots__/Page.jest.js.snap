// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Page should match previous snapshot 1`] = `
<div
  className="row"
  id="main-window"
>
  <div>
    hello
  </div>
</div>
`;
